content/open-source/herman.rst

```diff
@@ -0,0 +1,27 @@
+public: yes
+location: open-source
+grid_template: true
+image:
+  - src: 'blog/2016/coachhub/feature.jpg'
+project:
+  - name: 'Herman'
+    tagline: 'Automated Style Guides'
+    source: 'https://github.com/oddbird/sassdoc-theme-herman'
+    owner: yes
+    status: 'pre-release'
+contributors:
+  - author: 'oddbird'
```

Comment:
This doesn't look great. Could we at least change to `OddBird`? Even better, also link to `/`.

![screenshot 2017-03-24 16 31 26](https://cloud.githubusercontent.com/assets/552316/24312519/5e62f37a-10af-11e7-988b-0ac99ee67ca5.jpg)